tests/repro_tests/frontend/descriptor/test_descriptor_reproducibility_base.py

```diff
@@ -0,0 +1,36 @@
+""""Reproducibility tests for descriptors.
+
+Authors: Ayush Baid
+"""
+import unittest
+from pathlib import Path
+
+import numpy as np
+
+import gtsfm.utils.io as io_utils
+from gtsfm.common.image import Image
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.descriptor.descriptor_base import DescriptorBase
+from gtsfm.frontend.descriptor.sift import SIFTDescriptor
+from gtsfm.frontend.detector.detector_base import DetectorBase
+from gtsfm.frontend.detector.dog import DoG
+from tests.repro_tests.test_repro_base import ReproducibilityTestBase
+
+# defining the path for test data
+TEST_DATA_ROOT_PATH = Path(__file__).resolve().parent.parent.parent.parent / "data"
+IMG_PATH = TEST_DATA_ROOT_PATH / "set1_lund_door" / "images" / "DSC_0001.JPG"
+
+
+class DescriptorReproducibilityTestBase(ReproducibilityTestBase, unittest.TestCase):
+    def setUp(self, descriptor: DescriptorBase = SIFTDescriptor()) -> None:
+        super().setUp()
+        detector: DetectorBase = DoG()
```

Comment:
could you note in the docstring that we are using DoG as a fixed detector, for the sake of these tests? I think that's important to highlight.

Reply:
marking as resolved, thanks.